src/main/java/org/opentripplanner/api/mapping/LegMapper.java

```diff
@@ -98,12 +98,12 @@ public ApiLeg mapLeg(
       api.agencyBrandingUrl = agency.getBrandingUrl();
 
       var route = domain.getRoute();
-      api.route = route.getLongName();
+      api.route = route.getLongName().toString();
       api.routeColor = route.getColor();
       api.routeType = domain.getRouteType();
       api.routeId = FeedScopedIdMapper.mapToApi(route.getId());
       api.routeShortName = route.getShortName();
-      api.routeLongName = route.getLongName();
+      api.routeLongName = route.getLongName().toString();
```

Comment:
`getLongName` is nullable so you need to create a utility method to deal with that.

Reply:
Good point! Any suggestions where to put this utility method?

Reply:
https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/api/mapping/I18NStringMapper.java#L8

You can add a second version, with null locale